dm/config/subtask.go

```diff
@@ -121,10 +122,6 @@ func (db *DBConfig) Decode(data string) error {
 
 // Adjust adjusts the config.
 func (db *DBConfig) Adjust() {
```

Comment:
Can we remove it now?

Reply:
not sure, maybe we could use in future 🤔

Reply:
ah some many reference, I prefer leave it here in case of we need it in future